Homework 1

hamosapience opened the pull request:
Опоздал к дедлайну, так что комментариев и не жду :-) 

UPD: https://miro.com/app/board/o9J_lcJRvy8=/?moveToWidget=3458764524548750754&cot=14

hamosapience committed:
Homework 1

davydovanton commented:
привет! а можешь на сайт или на почту скинуть ссылку на репозиторий? я иначе не смогу отметить, что ты сделал дз, а потом пингани - я посмотрю + засчитаю первое дз

hamosapience commented:
> а можешь на сайт или на почту скинуть ссылку на репозиторий

Готово. 
@davydovanton

davydovanton commented:
Привет, все посмотрел

На разборе удели пожалуйста внимание:

- как аудит лог аккаунтинга может работать
- разбору вариантов хранения цены тасков
- объединение асайна в создании и в вызове менеджером
- примеру создания бизнес цепочек
 - разбору нейминга, а именно деталий имплементации в названиях и названия событий

hamosapience committed:
update scheme according to feedback:
- task fee and reward now stored in finance domain
- CUD event is used for notifying finance domain
- Task-related BE events names changed from *created to *added to avoid technical connotations / pushing towards certain technical solutions

hamosapience commented:
Поправил что нашёл по результатам разбора домашек.
